diff --git a/runtime/robonomics/src/lib.rs b/runtime/robonomics/src/lib.rs
index 4b38bfa5..8d44d43b 100644
--- a/runtime/robonomics/src/lib.rs
+++ b/runtime/robonomics/src/lib.rs
@@ -708,9 +708,11 @@ impl frame_support::traits::OnRuntimeUpgrade for InitMigrationStorage {
             .filter_map(|v| pallet_session::Pallet::<Runtime>::load_keys(&v).map(|k| (v, k)))
             .collect();
 
+        /*
         <Runtime as pallet_session::Config>::SessionHandler::on_genesis_session::<
             <Runtime as pallet_session::Config>::Keys,
         >(&queued_keys);
+        */
 
         Validators::<Runtime>::put(initial_validators_0);
         QueuedKeys::<Runtime>::put(queued_keys);
@@ -953,10 +955,12 @@ where
     I: frame_support::traits::ExecuteBlock<Block>,
 {
     fn execute_block(block: Block) {
+        /*
         if pallet_aura::Authorities::<T>::get().len() > 0 {
             // Execute block with aura pallet
             cumulus_pallet_aura_ext::BlockExecutor::<T, I>::execute_block(block);
         } else {
+        */
             // Workaround because we don't have authorities
             use sp_consensus_aura::digests::CompatibleDigestItem;
             use sp_runtime::{traits::Header, RuntimeAppPublic};
@@ -969,7 +973,7 @@ where
             );
 
             I::execute_block(Block::new(header, extrinsics));
-        }
+        //}
     }
 }
 
diff --git a/scripts/build-runtime.sh b/scripts/build-runtime.sh
index 1ab1f455..9b1dddca 100755
--- a/scripts/build-runtime.sh
+++ b/scripts/build-runtime.sh
@@ -3,5 +3,5 @@
 export RUSTC_VERSION=1.88.0
 export PACKAGE=robonomics-runtime
 
-docker run --rm -it -e PACKAGE=$PACKAGE -e BUILD_OPTS=$1 -v $PWD:/build \
+docker run --rm -it -e PACKAGE=$PACKAGE -e BUILD_OPTS='--features on-chain-release-build' -v $PWD:/build \
     -v $TMPDIR/cargo:/cargo-home paritytech/srtool:$RUSTC_VERSION
